consumer/src/app/(purchase)/produtores/page.tsx

```diff
@@ -1,110 +1,110 @@
 "use client";
 
-import { Catalog, Farm, fetchCatalogs } from "@consumer/app/_actions/fetch-catalogs";
-import Image from "next/image";
-import Link from "next/link";
 import { useEffect, useState } from "react";
-import { useInView } from 'react-intersection-observer';
+import { useInView } from "react-intersection-observer";
 import { Cycle, fetchCycles } from "../../_actions/fetch-cycles";
 import RedirectCart from "@consumer/app/_components/redirectCart";
-export default function Produtores() {
+import ProducerCard from "./components/ProducerCard";
+import { searchCatalogs } from "@consumer/app/_actions/search-catalogs";
+import { CatalogDTO } from "@shared/domain/dtos/catalog-dto";
+import { useHandleError } from "@shared/hooks/useHandleError";
 
-  const [cycles, setcycle] = useState([] as Cycle[]);
-  const [cycleId, setCycleId] = useState('' as string);
+export default function Produtores() {
+  const [cycles, setCycles] = useState([] as Cycle[]);
+  const [cycleId, setCycleId] = useState("" as string);
   const [producers, setProducers] = useState([] as any[]);
   const [page, setPage] = useState(1 as number);
   const [isLoading, setIsLoading] = useState(true);
-  const { ref, inView } = useInView()
-
+  const { ref, inView } = useInView();
+  const { handleError } = useHandleError();
 
   useEffect(() => {
     (async () => {
-      setcycle(await fetchCycles());
+      const cycles = await fetchCycles();
+      setCycles(cycles);
     })();
   }, []);
-  
+
   const searchProducers = async () => {
+    const typeCycle = process.env.NEXT_PUBLIC_ENV
+      ? process.env.NEXT_PUBLIC_ENV === "dev" ||
```

Comment:
Validar se realmente é necessário ter essa variável criada, tendo-se a existência do NODE_ENV que nos mostra se o ambiente que estamos é prod ou dev. 
Pelo que entendi você está validando do ambiente de homolog, faz uma análise se realmente é necessário, se for, ignora meu comentário, se não, cabe um ajuste nisso.